crates/bevy_window/src/window.rs

```diff
@@ -213,6 +213,14 @@ pub struct Window {
     ///
     /// - iOS / Android / Web: Unsupported.
     pub window_theme: Option<WindowTheme>,
+    /// Sets the window's visibility.
+    ///
+    /// If `false`, this will hide the window. If `true`, this will show the window.
```

Comment:
What does "hide" and "show" mean exactly, is it like the window doesn't even appear in the task bar if it's hidden, or does it minimize it? When setting to true, if it's minimized, does it maximizes it, what about if it's hidden behind another window, does it bring it into focus?

This should be more clearly documented. I know it's not documented on Winit's side but still.

Reply:
I copied the comment from the winit docs for `set_visible`.

This just makes the window visible or not visible, it doesn't touch the minimized/maximized status, it's only about the visibility of the window. If I'm being honest, I don't really understand what is not clear here. I don't mind a better suggestion but I just don't know what to change or add here.

Reply:
LGTM thanks